component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateAuthenticator.java

```diff
@@ -545,6 +545,20 @@ private void validateUsingSubject(String subject, AuthenticationContext authenti
             addOrValidateCertificate(subject, authenticationContext, data, claims, cert);
         }
     }
+    /**
+     * check and validate the tenant domain of the user.
+     *
+     * @param userName               user name of the authenticated user.
+     */
+    private String getAuthenticUserName(String userName){
+        String authenticUser = null;
```

Comment:
Please check the formatting. Please check in other places as well.

Reply:
fixed